apps/emqx_auth_http/src/emqx_auth_http.appup.src

```diff
@@ -1,26 +1,32 @@
-%% -*-: erlang -*-
-
+%% -*- mode: erlang -*-
 {VSN,
-  [
-    {"4.3.2", [
-     {apply, {application, stop, [emqx_auth_http]}},
-     {load_module, emqx_auth_http_app, brutal_purge, soft_purge,[]},
-     {load_module, emqx_auth_http_cli, brutal_purge, soft_purge,[]}
-    ]},
-    {<<"4.3.[0-1]">>, [
-     {restart_application, emqx_auth_http}
-    ]},
-    {<<".*">>, []}
-  ],
-  [
-    {"4.3.2", [
-     {apply, {application, stop, [emqx_auth_http]}},
-     {load_module, emqx_auth_http_app, brutal_purge, soft_purge,[]},
-     {load_module, emqx_auth_http_cli, brutal_purge, soft_purge,[]}
-    ]},
-    {<<"4.3.[0-1]">>, [
-     {restart_application, emqx_auth_http}
-    ]},
-    {<<".*">>, []}
-  ]
-}.
+  [{"4.3.2",
+    [{load_module,emqx_auth_http_sup,brutal_purge,soft_purge,[]},
+     {load_module,emqx_auth_http,brutal_purge,soft_purge,[]},
+     {load_module,emqx_acl_http,brutal_purge,soft_purge,[]},
+     {apply,{application,stop,[emqx_auth_http]}},
```

Comment:
@tigercl  why do we apply application stop instead of using the `restart_application` instruction ?